datasets/pyproject.toml

```diff
@@ -144,6 +144,8 @@ exclude = [
     "venv",
     "proto",
 ]
+[tool.ruff.per-file-ignores]
```

Comment:
```suggestion
]

[tool.ruff.per-file-ignores]
```

Reply:
@adam-narozniak can we keep the ruff config synced across all projects (framework/baselines/datasets)?

Reply:
Actually, when there the
```__all__ = [
    "IidPartitioner",
    "Partitioner",
]```
is added, there's no longer any problems (it was before since the imports were not used and formatting removed them automatically)

Reply:
@adam-narozniak does this mean we can remove the F401-related configs from .flake8 / pyproject.toml?

Reply:
Yes, I forgot about the flake.
I'll remove the  F401-related configs from .flake8
I'll also check if in that case, I can remove the whole l.flake8 and if it's automatically based on the one for flower